index.html

```diff
@@ -0,0 +1,35 @@
+<!-- Boilerplate 1-10-->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
+    <meta name="viewport" content="width=device-width", initial-scale="1.0">
+    <title>Language Switcher</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<!-- Main-section 14-31-->
+
+<body>
+    <div class="container">
+        <div class="language-section">
+            <button class="button active" data-language="english">English</button>
+            <button class="button" data-language="french">French</button>
+            <button class="button" data-language="german">German</button>
+        </div>
+        <div class="content">
+            <h1 class="title">
+                Translating static text.
+            </h1>
+            <p class="description">
+                Translating static text.
+            </p>
+        </div>
+
+    </div>
+    <script src="index.js"></script>
```

Comment:
Move this to the `<head>` section with a [defer](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer).
```suggestion
    <script defer src="index.js"></script>
```